### ISSUE TITLE
[Suggestion] No point to iterate, parse and diff stuff when config.showArrivalOnDifference is null

### ISSUE THREAD

**Issue by avioli** [BlinkTagInc/gtfs-to-html#153]
https://github.com/BlinkTagInc/gtfs-to-html/blob/9555adbbb9c1f571a6f47e580abfd241cc4d3317/lib/utils.js#L613

Given that the config is not going to change during a run of the tool, this check could be moved to the top of the function and immediately return.

```diff
const duplicateStopsForDifferentArrivalDeparture = (
  stopIds,
  timetable,
  config
) => {
+   if (config.showArrivalOnDifference === null) {
+     return stopIds;
+   }
+ 
  for (const trip of timetable.orderedTrips) {
    for (const stoptime of trip.stoptimes) {
      const timepointDifference = fromGTFSTime(stoptime.departure_time).diff(
        fromGTFSTime(stoptime.arrival_time),
        'minutes'
      );

      if (
-         config.showArrivalOnDifference === null ||
        timepointDifference < config.showArrivalOnDifference
      ) {
        continue;
      }

      const index = stopIds.indexOf(stoptime.stop_id);
      if (index === 0 || index === stopIds.length - 1) {
        continue;
      }

      if (
        stoptime.stop_id === stopIds[index + 1] ||
        stoptime.stop_id === stopIds[index - 1]
      ) {
        continue;
      }

      stopIds.splice(index, 0, stoptime.stop_id);
    }
  }

  return stopIds;
};
```

**Comment by avioli** [comment]
Let me know if you would like a PR.

**Comment by brendannee** [comment]
Great suggestion!

I implemented this and released an updated version 2.5.1 with this and [added you as contributor](https://github.com/BlinkTagInc/gtfs-to-html/blob/master/package.json).

Let me know if you see other ways to optimize things or improvements to make.
